modules/ensemble/lib/framework/data_context.dart

```diff
@@ -464,9 +465,24 @@ class NativeInvokable extends ActionInvokable {
       'formatter': () => Formatter(),
       'utils': () => _EnsembleUtils(),
       'device': () => Device(),
+      'version': () => getEnsembleVersion(),
```

Comment:
cache and mutex this. We'll be doomed if user put this in a loop. Less desirable but can also do this once at start up.

Reply:
I have added cache. It will cache on first run not at startup at the moment